### PR TITLE
[Tyr] Traveller profile

### PR DESCRIPTION
**Pull request by nonifier**
This PR does 3 things:
 - It fixes Tyr's endpoint `Tyr/<version>/instances/<coverage>/traveler_profiles` to display all traveller profiles.
 - It adds to a coverage all its `traveler_profiles` when querying `Tyr/<version>/instances/<coverage>`
 - it adds a new endpoint to Tyr's instance. To create a default coverage in DB using a **POST** on `Tyr/<version>/instances/<coverage>`

### CODE REVIEW

File: source/tyr/tyr/resources.py [hunk 1]
```diff
@@ -1441,7 +1441,7 @@ def check_resources(f):
         @wraps(f)
         def wrapper(*args, **kwds):
             tp = kwds.get('traveler_type')
-            if tp in acceptable_traveler_types:
+            if tp in acceptable_traveler_types or tp is None:
```
Comment (xlqian) [inline source/tyr/tyr/resources.py:1460]: lol

**Review by xlqian — APPROVED** [review]

**Comment by xlqian** [comment]
any tests? if it's too complicated... let it be :)

File: source/tyr/tests/integration/instance_test.py [hunk 2]
```diff
@@ -90,6 +90,13 @@ def test_get_instance(create_instance):
     assert resp[0]['id'] == create_instance
 
 
+def test_get_instance_with_traveler_profile(create_instance):
```
Comment (xlqian) [inline source/tyr/tests/integration/instance_test.py:93]: :+1:

**Review by xlqian — APPROVED** [review]

**Comment by nonifier** [comment]
@xlqian Thanks for the review, I have been lazy on this one :disappointed:, but promise, never again :) 
Tests are now part of commit https://github.com/CanalTP/navitia/pull/3270/commits/49114aa6a28beaafa07cfb39c63e612736932cf2

**Review by pbench — APPROVED** [review]

**Pull request merged**